TramsDataApi.Test/UseCases/GetEstablishmentURNsByRegionTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using FizzWare.NBuilder.PropertyNaming;
+using FizzWare.NBuilder;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using FluentAssertions;
+using TramsDataApi.CensusData;
+using TramsDataApi.DatabaseModels;
+using TramsDataApi.Factories;
+using TramsDataApi.Gateways;
+using TramsDataApi.RequestModels;
+using TramsDataApi.UseCases;
+using Xunit;
+
+namespace TramsDataApi.Test.UseCases
+{
+    public class GetEstablishmentURNsByRegionTests
+    {
+        private readonly GetEstablishmentsByRegion _useCase;
+        private readonly Mock<IEstablishmentGateway> _establishmentGateway;
+
+        private static readonly List<string> Regions = new List<string>()
+        {
+            "East",
+            "West"
+        };
+        private static readonly List<string> SingularRegion = new List<string>()
+        {
+            "East"
+        };
+
+        public GetEstablishmentURNsByRegionTests()
+        {
+            BuilderSetup.SetDefaultPropertyName(new RandomValuePropertyNamer(new BuilderSettings()));
+
+            _establishmentGateway = new Mock<IEstablishmentGateway>();
+            _useCase = new GetEstablishmentsByRegion(_establishmentGateway.Object);
+        }
+
+        [Fact]
+        public void GetEstablishmentURNsByRegion_ReturnsNull_WhenNoEstablishmentsAreFound()
+        {
+            _establishmentGateway.Setup(gateway => gateway.GetURNsByRegion(Regions)).Returns(() => null);
+
+            _useCase.Execute(Regions).Should().BeNull();
```

Comment:
Small thing: as discussed yesterday perhaps it'd be better to return an empty list.

Reply:
Ah yeah, thanks!